packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -0,0 +1,157 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+/* tslint:enable:no-unused-variable */
+
+import * as PropTypes from 'prop-types';
+import {
+  BaseComponent,
+  autobind
+} from '../../Utilities';
+import { ScrollablePane } from '../../ScrollablePane';
+import { IStickyProps } from './Sticky.Props';
+import * as stylesImport from './Sticky.scss';
+const styles: any = stylesImport;
+
+export interface IStickyState {
+  isStickyTop: boolean;
+  isStickyBottom: boolean;
+  placeholderHeight?: number;
+}
+
+export interface IStickyContext {
+  subscribe: PropTypes.Requireable<Function>;
+  addStickyHeader: PropTypes.Requireable<Function>;
+  removeStickyHeader: PropTypes.Requireable<Function>;
+  addStickyFooter: PropTypes.Requireable<Function>;
+  removeStickyFooter: PropTypes.Requireable<Function>;
+}
+
+export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
+  public static contextTypes: IStickyContext = {
+    subscribe: PropTypes.func,
+    addStickyHeader: PropTypes.func,
+    removeStickyHeader: PropTypes.func,
+    addStickyFooter: PropTypes.func,
+    removeStickyFooter: PropTypes.func
+  };
+
+  public refs: {
+    root: HTMLElement;
+    placeholder: HTMLElement;
+  };
+
+  public context: {
+    subscribe: (handler: Function) => void;
+    addStickyHeader: (sticky: Sticky) => void;
+    removeStickyHeader: (sticky: Sticky) => void;
+    addStickyFooter: (sticky: Sticky) => void;
+    removeStickyFooter: (sticky: Sticky) => void;
```

Comment:
These should not be individual fields under `context`. This means that anyone would collide on a field named `subscribe`. You should make it look this this:

```ts
public context: {
    scrollablePane: IScrollablePane;
}
```